client/src/components/leaderboard.tsx

```diff
@@ -15,16 +19,19 @@ const Leaderboard = ({scores}:LeaderboardProps) => {
 
     return (
         <div className={"leaderboard"}>
-            {scores.map(score => <Score key={score.name} name={score.name} score={score.score} />)}
+            {scores.map(score => <Score key={score.name} name={score.name} score={score.score} pos={scores.indexOf(score) + 1} />)}
```

Comment:
I'm pretty sure you can do it like this instead: `{scores.map((score, index) => <Score key={score.name} name={score.name} score={score.score} pos={index + 1} />)}`. It looks a little bit cleaner imo :)

Reply:
oh yea, thats much better! Thanks